src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsList.tsx

```diff
@@ -147,6 +176,17 @@ const ConditionsList: React.FC<IConditionsListProps> = (props) => {
                                 ),
                                 selectedTarget: `#${LIST_KEY}`
                             });
+                        },
+                        buttonProps: {
+                            customStyles: {
+                                root: {
+                                    [primaryTextClassName]: {
+                                        fontStyle: condition.isUnlabeled
```

Comment:
similar to my previous comment about styling, can we move this out of the styles file and use a styling from there here?